zombies.js

```diff
@@ -137,7 +193,26 @@
  * @param {Item/Weapon/Food} item   The item to discard.
  * @return {boolean} true/false     Whether player was able to remove item from pack.
  */
+discardItem(item) {
+  let index = this._pack.indexOf(item);
+
+  if(index === -1) {
+    console.log(item.itemName + " not found in pack");
+    return false;
+  } else {
+    this._pack.splice(index, 1);
+
+    console.log(this.name+ " discards " + item.itemName);
+    return true;
+  }
+}
 
+checkPack() {
+  let pack = this.getPack().map(function(element, index, array) {
+    return element.itemName;
+  }).toString();
```

Comment:
The `.toString()` isn't really needed here. The `console.log` will do that for you.